utils.go

```diff
@@ -1,65 +1,206 @@
 package main
 
 import (
+	"crypto/tls"
 	"image"
 	"io"
 	"log"
+	"net/http"
 	"os"
 	"strings"
 
 	"encoding/base64"
 	"image/png"
 
 	"github.com/lxn/walk"
-	"github.com/tink-ab/tempfile"
 )
 
 // Create and write profile file into templateToFile folder
 func createLanguageFile(currentDir, translationLanguage, languageFileName string) error {
 	// create and open file
 	languageFile, err := os.Create(currentDir + "\\" + languageFileName)
 	if err != nil {
-		walk.MsgBox(windowMsgBox, "Error", "Unable to create the language file, please contact your local support.", walk.MsgBoxOK)
-		log.Fatal("Failed creating language file: ", err)
+		viewErrorAndExit("Unable to create the language file.", err.Error())
 		return err
+	} else {
+		// close file
+		defer languageFile.Close()
+		// write the template into the new file
+		_, err = io.Copy(languageFile, strings.NewReader(translationLanguage))
+		if err != nil {
+			viewErrorAndExit("Unable to write into language file.", err.Error())
+			return err
+		} else {
+			addNewLinesToDebug("Language file successfully created.")
+			return nil
+		}
 	}
-	// close file
-	defer languageFile.Close()
-	// write the template into the new file
-	_, err = io.Copy(languageFile, strings.NewReader(translationLanguage))
-	if err != nil {
-		walk.MsgBox(windowMsgBox, "Error", "Unable to write into language file, please contact your local support.", walk.MsgBoxOK)
-		log.Fatal("Failed writing language constant to file: ", err)
-		return err
+}
+
+func addNewLinesToDebug(mytxt string) {
+	if debug {
+		log.Print(mytxt)
+		myTxtC := chunks(mytxt, 60)
+		myCt := debugTxt.Text()
+		debugTxt.SetText(myCt + myTxtC + "\r\n")
+		numlines := strings.Count(debugTxt.Text(), "\r\n")
+		debugHeight := (13 * numlines) + 13
+		if debugHeight <= 240 {
+			debugHeight = 240
+		}
+		debugTxtsize := walk.Size{Width: 400, Height: debugHeight}
+		debugTxt.SetMinMaxSize(walk.Size(debugTxtsize), walk.Size(debugTxtsize))
+	} else {
+		log.Print(mytxt)
 	}
-	log.Print("Language file successfully created.")
-	return nil
 }
 
+// Chunk large string at x charaters
+// Source: https://stackoverflow.com/a/61469854
+func chunks(s string, chunkSize int) string {
```

Comment:
Could you rename that function using usual convention for function (verbNoun) ? Perhaps `chunkLargeStrings`

In that function, a variable `chunks` has same name that the function `chunks`.

Reply:
Done